README.md

```diff
@@ -1,2 +1,88 @@
 # content-hub-one-nextjs-starterkit
-The Content Hub ONE NextJS starterkit will contain helping functionalities such as handling GraphQL JSON output conversion to HTML for Rich Text, Media Fields and References. Also it will contain a small example implementation that Developers can use to start their customer projects on.
+The Content Hub ONE NextJS starterkit contains helping functionalities such as handling GraphQL JSON output conversion to HTML for Rich Text, Media Fields and References. It also contains a small example implementation that Developers can use to start their customer projects on.
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+More Information about Content Hub ONE including Video Tutorials you find in the [Developer Portal]( https://developers.sitecore.com/content-management/content-hub-one).
+
+## Prerequisites
```

Comment:
Are tenants created in https://portal.sitecorecloud.io/? I seem to no longer be a part of an organization so I cannot create an instance. Meaning, I cannot fully go through the starter kit right now.

Reply:
We cannot create tenants on our own. But you can run the app here in the repo against the Content Hub ONE instance. As you are organization admin you should see the CH! instance.